src/libpmem2/config_windows.c

```diff
@@ -141,3 +141,19 @@ pmem2_config_get_file_size(const struct pmem2_config *cfg, size_t *size)
 
 	return 0;
 }
+
+/*
+ * pmem2_config_get_alignment -- get alignment from the system info
+ */
+int
+pmem2_config_get_alignment(const struct pmem2_config *cfg, size_t *alignment)
```

Comment:
Do we need to split it to windows and linux implementation

You can use Mmap_alligment variable (it is set to Page size on linux, and dwAllocationGranularity on windows)

Reply:
I think we need both versions, just because of Device DAX.